yarn-project/key-store/src/test_key_store.ts

```diff
@@ -81,8 +88,8 @@ export class TestKeyStore implements KeyStore {
    */
   public getAccounts(): Promise<AztecAddress[]> {
     const allMapKeys = Array.from(this.#keys.keys());
-    // We return account addresses based on the map keys that end with '-nsk_m'
-    const accounts = allMapKeys.filter(key => key.endsWith('-nsk_m')).map(key => key.split('-')[0]);
+    // We return account addresses based on the map keys that end with '-nsk_m-account'
+    const accounts = allMapKeys.filter(key => key.endsWith('-nsk_m-account')).map(key => key.split('-')[0]);
```

Comment:
Would rather that we simply use a regex it to match instead here or adding extra things to the names, it is brittle. `nsk_m$` should help you. Otherwise take a look at the extension suggested above.